src/routes/users/deleteUser.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  RawReplyDefaultExpression,
+  RawRequestDefaultExpression,
+  RawServerDefault,
+  RouteHandler,
+  RouteOptions,
+} from "fastify";
+
+import { User } from "../../types/User";
+import UserModel from "../../models/user";
+import { ErrorSchema } from "../../types/ErrorSchema";
+import ErrorSchemaJson from "../../schemas/ErrorSchema.json";
+import { UserRequestParams } from "../../types/UserRequestParams";
+import UserRequestParamsSchema from "../../schemas/UserRequestParams.json"
+
+type Reply = ErrorSchema;
+type DeleteUserRoute = {
+  Body: User;
+  Params: UserRequestParams;
+  Reply: Reply;
+};
+
+const url = "/user/:id";
+
+const handler: RouteHandler<DeleteUserRoute> = async (req, rep) => {
+  try {
+    const userId = req.params.id;
```

Comment:
Can we destructure id instead with an alias?
```suggestion
const {id:userId} = req.params;
```

Reply:
nitpick 😸